src/components/ApplnBar.js

```diff
@@ -102,7 +102,7 @@ const ApplnBar = () => {
         <DynamicTabs
           component1={<AddCropCycle />}
           component2={'c2'}
-          component3={'c3'}
+          component3={<FarmTimeline />}
```

Comment:
Please rename the file to `FieldTimeline`. We are using `field` instead of `farm` everywhere.

Reply:
Actually, this should be `CropCycleTimeline`.